index.html

```diff
@@ -39,5 +40,144 @@ <h2>LET'S CONNECT</h2>
         </ul>
         
     </section>
-</body>
-</html>
\ No newline at end of file
+    <section class="project-container">
+        <div class="project">   
+        <img src="SnapshootPortfolio.png" alt="picture identity of a lady">
+        <h2>Tonic</h2>
+        <ol>
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
```

Comment:
I see that you have gray space on right, it would be better if you try to remove it  to be the same as the Figma design 
![image](https://user-images.githubusercontent.com/61790587/166077241-2c80099f-5d8f-44f2-b384-be105492a54d.png)

Reply:
ok i have resolved this issue thanks